spec/features/user_can_adjust_quantity_of_item_in_cart_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.